libraries/core_libs/storage/src/storage.cpp

```diff
@@ -90,7 +97,10 @@ void DbStorage::rebuildColumns(const rocksdb::Options& options) {
   std::transform(column_families.begin(), column_families.end(), std::back_inserter(descriptors), [](const auto& name) {
     return rocksdb::ColumnFamilyDescriptor(name, rocksdb::ColumnFamilyOptions());
   });
-  checkStatus(rocksdb::DB::Open(options, db_path_.string(), descriptors, &handles, &db));
+  rocksdb::DB* db_ptr = nullptr;
+  checkStatus(rocksdb::DB::Open(options, db_path_.string(), descriptors, &handles, &db_ptr));
```

Comment:
I think db_ptr should point to heap? Does it should delete db_ptr later?

Reply:
db_ptr points to nothing, then `rocksdb::DB::Open` will assign it object and destructor of unique_ptr will delete it when we return from function